client/src/componentsStyles/PagesStyle.jsx

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components';
+
+const PageButton = styled.button`
+  display: inline-block;
+  background-color: transparent;
+  color: rgb(0, 132, 137);
+  border-width: initial;
+  border-style: none;
+  border-color: initial;
+  border-image: initial;
+  font: inherit inherit inherit inherit inherit inherit inherit inherit inherit;
+  text-decoration: none;
+`;
+
+const ArrowButtons = styled.div`
+  -webkit-box-pack: center;
+  -webkit-box-align: center;
+  align-items: center;
+  background-color: transparent;
+  box-shadow: rgb(0, 132, 137) 0px 0px 0px 1px inset;
+  color: rgb(0, 132, 137);
+  display: flex;
+  height: 32px;
+  justify-content: center;
+  width: 32px;
+  border-radius: 50%;
+`;
+
+const NumberInButton = styled.div`
+  width: 32px;
+  height: 32px;
+  line-height: 32px;
+  text-align: center;
+  border-radius: 16px;
+`;
+const SelectedButton = styled.button`
+  display: inline-block;
+  background-color: transparent;
+  color: rgb(255, 255, 255);
+  border-width: initial;
+  border-style: none;
+  border-color: initial;
+  border-image: initial;
+  font: inherit inherit inherit inherit inherit inherit inherit inherit inherit;
```

Comment:
Seems like a ton of stuff that you likely don't need here. Maybe try removing it and see if it makes a difference?